pdc/apps/utils/__init__.py

```diff
@@ -16,5 +18,20 @@
     'stomp': StompMessenger,
 }
 
-# init messenger
-messenger = MESSENGERS.get(settings.MESSAGE_BUS['MLP'], DummyMessenger)()
+
+def _init_messenger():
+    mpl = settings.MESSAGE_BUS['MLP']
```

Comment:
I think MPL stands for Messaging Library Provider. `mpl` looks like a typo.

Reply:
Yes, it's typo.